docs-src/src/layouts/index.scss

```diff
@@ -260,8 +260,8 @@ code {
   }
 
   .rev-Menu-item--text {
-    color: $color-white-60;
     h5 {
+      color: $color-black-80;
```

Comment:
@kamrinkennedy Instead of updating the `h5` color, lets update the navigation labels to an `h3` tag and add the following styles to the navigation labels

- color: $color-white
- padding: 3px
- border-bottom: 1px solid $color-white

Navigation labels

<img width="298" alt="Screen Shot 2021-01-27 at 4 01 31 PM" src="https://user-images.githubusercontent.com/49654454/106060912-47387d00-60ba-11eb-978a-f64187c62a4a.png">


Here is what it looks like from the element inspector 

<img width="289" alt="Screen Shot 2021-01-27 at 3 57 50 PM" src="https://user-images.githubusercontent.com/49654454/106060758-13f5ee00-60ba-11eb-9399-63ea646cb642.png">